src/components/Avatar/Avatar.tsx

```diff
@@ -56,10 +59,13 @@ export const Avatar = <
 
   const nameStr = name?.toString() || '';
   const initials = getWholeChar(nameStr, 0);
+  const className = ['str-chat__avatar', `str-chat__avatar--${shape}`, customClassName]
+    .join(' ')
+    .trim();
```

Comment:
This looks like recurring "issue" across the SDK so I suggest we either use [`classnames`](https://www.npmjs.com/package/classnames) or create our own lightweight implementation of it so we can get rid of patterns like this one or this one:
```
<div className={`${chatClass} ${channelClass} str-chat__channel ${theme}`}>
```

Reply:
https://github.com/GetStream/stream-chat-react/issues/1477

Reply:
I am not sure, whether this should be inside the scope of the theming v2 epic. It may introduce bugs and is not necessarily related to deliver correctly  functioning theming v2.

Reply:
no, not in the scope of v2 for sure.